test/FTL.sql

```diff
@@ -0,0 +1,152 @@
+CREATE TABLE ftl
+(
+    id    INTEGER PRIMARY KEY NOT NULL,
+    value BLOB                NOT NULL
+);
+
+CREATE TABLE counters
+(
+    id    INTEGER PRIMARY KEY NOT NULL,
+    value INTEGER             NOT NULL
+);
+
+CREATE TABLE network
+(
+    id         INTEGER PRIMARY KEY NOT NULL,
+    ip         TEXT                NOT NULL,
+    hwaddr     TEXT                NOT NULL,
+    interface  TEXT                NOT NULL,
+    name       TEXT,
+    firstSeen  INTEGER             NOT NULL,
+    lastQuery  INTEGER             NOT NULL,
+    numQueries INTEGER             NOT NULL,
+    macVendor  TEXT
+);
+
+CREATE TABLE queries
+(
+    id        INTEGER PRIMARY KEY AUTOINCREMENT,
+    timestamp INTEGER NOT NULL,
+    type      INTEGER NOT NULL,
+    status    INTEGER NOT NULL,
+    domain    TEXT    NOT NULL,
+    client    TEXT    NOT NULL,
+    forward   TEXT
+);
+
+CREATE INDEX idx_queries_timestamps ON queries (timestamp);
+
+INSERT INTO ftl
+VALUES
+    -- Version
+    (0, 3),
+    -- Last query timestamp
+    (1, 177180),
+    -- First counter timestamp
+    (2, 0);
+
+INSERT INTO counters
+VALUES
+    -- Total queries
+    (0, 5980376),
+    -- Blocked queries
+    (1, 19382);
+
+INSERT INTO network
+VALUES (1, '10.1.1.1', '00:00:00:00:00:00', 'eth0', 'gateway', 1546832160, 1547002023, 6, '');
```

Comment:
We should have a non-empty vendor string so we can test against it.
```suggestion
VALUES (1, '10.1.1.1', '00:00:00:00:00:00', 'eth0', 'gateway', 1546832160, 1547002023, 6, 'Fantasy Devices Inc.');
```